### PR TITLE
ActiveCampaign documentation update

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Updated ActiveCampaign docs to explicitly specify that the `screen` and `track` calls must have only alphanumeric characters in the event name. Added a FAQ for this as well.
> Reordered the doc to include the following order of sections: Identify > Page > Screen > Track.
> Content updates to standardize the doc.

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Review by yashasvibajpai — APPROVED** [review]

File: destinations/marketing/activecampaign.mdx [hunk 1]
```diff
@@ -26,111 +26,52 @@ Before configuring your source and destination on the RudderStack, please check
...
-
-If you want to set a value for the `screen` and `track` type event, you need to pass the `name` property in the event properties, as shown in the above example.
-
-</div>
+- **Event Key:** This value is unique to your ActiveCampaign account. To obtain the event key, go to your ActiveCampaign account and navigate to **Settings** > **Tracking** > **Event Tracking**.
+- **ActID:** This value is unique to your ActiveCampaign account.  Go to **Settings** > **Tracking** > **Event Tracking API**. You will find the **actid** listed here.
 
 ## Identify
 
 The `identify` call lets you associate a user with their actions and capture all the relevant traits about them. This information includes unique `userid` as well as any optional information such as name, email address, etc.
```
Comment (ishachopra0401) [inline destinations/marketing/activecampaign.mdx:56]:
```suggestion
The `identify` call lets you associate a user with their actions and captures all the relevant traits about them. This information includes unique `userid` as well as any optional information such as name, email address, etc.
```

File: destinations/marketing/activecampaign.mdx [hunk 2]
```diff
@@ -26,111 +26,52 @@ Before configuring your source and destination on the RudderStack, please check
...
   })
 ```
 
 In the above snippet, RudderStack captures relevant information about the user such as the `userId` as well as the associated traits such as `email`, `phone`, and the name (`firstName` and `lastName`) of that user.
 
 <div class="infoBlock">
 
-The `email` trait is a mandatory trait for mapping a user to ActiveCampaign. If a user already exists, the new values will be updated for that user.
-
+The <code class="inline-code">email</code> trait is a mandatory trait for mapping a user to ActiveCampaign. If a user already exists, the new values will be updated for that user.
```
Comment (ishachopra0401) [inline destinations/marketing/activecampaign.mdx:74]: >If a user already exists, the new values will be updated for that user.
bit confusing, does it mean that if a user already exists, the new 'email' value will be updated for that user?

File: destinations/marketing/activecampaign.mdx [hunk 3]
```diff
@@ -140,34 +81,32 @@ You can associate a user with custom tags by passing in the `tags` trait, as sho
...
     tags: ["Returning User", "Coupon Used"],
   })
-
 ```
 
 <div class="infoBlock">
 
-The `tags` property should contain an array of tags which you want to associate with the user. If any tag is already created in the destination previously, RudderStack will automatically skip the creation of that tag.
-
+The <code class="inline-code">tags</code> property should contain an array of tags which you want to associate with the user. If any tag is already created in the destination previously, RudderStack will automatically skip the creation of that tag.
```
Comment (ishachopra0401) [inline destinations/marketing/activecampaign.mdx:94]:
```suggestion
The <code class="inline-code">tags</code> property should contain an array of tags which you want to associate with the user. If any tag is already created in the destination previously, RudderStack will override the tag information with the new one.
```

Does it mean this?

File: destinations/marketing/activecampaign.mdx [hunk 4]
```diff
@@ -179,14 +118,12 @@ rudderanalytics.identify(
 
 <div class="infoBlock">
 
-The `fieldInfo` property contains the value of the field information that you want to store for that contact. For using this feature, you have to create the fields from your ActiveCampaign dashboard \(For example - `Interest, Country`\) before passing in the values for the given user.
-
+The <code class="inline-code">fieldInfo</code> property contains the value of the field information that you want to store for that contact. For using this feature, you have to create the fields from your ActiveCampaign dashboard \(For example - Interest, Country\) before passing the values for the given user.
```
Comment (ishachopra0401) [inline destinations/marketing/activecampaign.mdx:121]:
```suggestion
The <code class="inline-code">fieldInfo</code> trait contains the value of the custom field that you want to store for that contact. For using this feature, you have to create the custom fields from your ActiveCampaign dashboard \(For example - Interest, Country\) before passing the values for the given user.
```

We have used `fieldInfo` **trait** above. Can 'trait' and 'property' be used interchangeably?

File: destinations/marketing/activecampaign.mdx [hunk 5]
```diff
@@ -220,9 +157,81 @@ rudderanalytics.identify(
 
 <div class="infoBlock">
 
-For associating a contact to any field, you need to create the list from ActiveCampaign and use the `id` parameter for mapping that contact.
+For associating a contact to any field, you need to create the list from ActiveCampaign and use the <code class="inline-code">id</code> parameter for mapping that contact.
 </div>
 
+## Page
+
+The `page` call allows you to record information whenever a user sees a web page, along with the associated optional properties of that page. This method must be called at least once per page load.
```
Comment (ishachopra0401) [inline destinations/marketing/activecampaign.mdx:165]:
```suggestion
The `page` call lets you record information whenever a user sees a web page, along with the associated optional properties of that page. This method must be called at least once per page load.
```

File: destinations/marketing/activecampaign.mdx [hunk 6]
```diff
@@ -220,9 +157,81 @@ rudderanalytics.identify(
...
+In the above sample, RudderStack captures the information related to the page being viewed, the URL `property` is used to whiltelist the website in the destination.
+
+<div class="infoBlock">
+
+A <code class="inline-code">page</code> call will only work if <strong>Site Tracking</strong> is enabled in ActiveCampaign. You can enable this setting by going to the <strong>Tracking</strong> tab in your ActiveCampaign settings page.
+</div>
+
+## Screen
+
+The `screen` method allows you to record whenever a user sees the mobile screen, along with any associated optional properties. This call is similar to the `page` call, but is exclusive to your mobile device.
```
Comment (ishachopra0401) [inline destinations/marketing/activecampaign.mdx:190]:
```suggestion
The `screen` method lets you record whenever a user sees the mobile screen, along with any associated optional properties. This call is similar to the `page` call, but is exclusive to your mobile device.
```

File: destinations/marketing/activecampaign.mdx [hunk 7]
```diff
@@ -220,9 +157,81 @@ rudderanalytics.identify(
...
+In the above snippet, RudderStack captures the information related to the `Product Purchased` event, along with any additional information about that event - in this case the name of the product.
+
+<div class="warningBlock">
+
+The <code class="inline-code">track</code> event name must contain only alphanumeric characters. ActiveCampaign rejects events with any special characters in the name.
+</div>
+
+<div class="infoBlock">
+
+If you want to set a value for the <code class="inline-code">screen</code> and <code class="inline-code">track</code> type event, you need to pass <code class="inline-code">name</code> in the event properties, as shown in the above example.
```
Comment (ishachopra0401) [inline destinations/marketing/activecampaign.mdx:226]: We have mentioned 'name' only in track-type event, not in screen event but say 'as shown in the above example'. Can be misleading..

File: destinations/marketing/activecampaign.mdx [hunk 8]
```diff
@@ -220,9 +157,81 @@ rudderanalytics.identify(
...
+</div>
+
+## FAQ
+
+### Can I use special characters in my `screen` and `track` event names?
+
+No, your `screen` and `track` event names must contain only alphanumeric characters. ActiveCampaign prescribes that the `screen` and `track` event names must contain only alphabets and numbers, and does not allow any special characters to be used.
```
Comment (ishachopra0401) [inline destinations/marketing/activecampaign.mdx:233]:
```suggestion
No, your `screen` and `track` event names must contain only alphanumeric characters. ActiveCampaign prescribes that the `screen` and `track` event names must contain only alphabets and numbers, and does not allow the usage of any special characters.
```

**Review by ishachopra0401 — CHANGES_REQUESTED** [review]

**Review by utsabc — APPROVED** [review]

**Review by ishachopra0401 — APPROVED** [review]

**Pull request merged**